app/src/main/java/edu/uc/it3048/mypokedex/SavedSightingsActivity.kt

```diff
@@ -23,7 +23,7 @@ class SavedSightingsActivity : AppCompatActivity() {
 
         // Allows for back arrow navigation
         val actionBar = supportActionBar
-        actionBar!!.title = "Saved Locations"
+        actionBar!!.title = "Saved Sightings"
```

Comment:
I changed this to "Saved Sightings" because the 2 header texts for this page ("Saved Locations" and "Pokemon Sighting Locations") seemed too different. Now they are more similar and look better on the page. ("Saved Sightings" and "Pokemon Sightings")

Reply:
Good... faster learning for the user.